device_independent_test/quantum_communicator.py

```diff
@@ -54,15 +54,15 @@ def __init__(self,backend):
     #           shot: number of shots to run
     # @returns  counts from running circuits or "NO MEASUREMENT" dictionary if
     #               there are no counts
-    def run_and_transmit(self,pre_operation,post_operations,shot):
+    def run_and_transmit(self, pre_operations, post_operations, shots):
         # compose a single circuit from the input operations
         size = max(post_operations[0].num_qubits,post_operations[1].num_qubits)
         qc = QuantumCircuit(size)
         qc += pre_operations
         qc += post_operations[0] + post_operations[1]
 
         # run circuit on backend
-        job = execute(qc, backend=self.devices[0], shots=shot)
+        job = execute(qc, backend=self.devices[0], shots=shots)
```

Comment:
this is fine I just prefer having different names as it distinguishes between the different arguments better